src/calculateRentalCost.js

```diff
@@ -27,8 +27,20 @@
  * @return {number}
  */
 
+const DAILY_PRICE = 40;
+
 function calculateRentalCost(days) {
-  // write code here
+  let discount = 0;
+
+  if (days >= 3) {
+    discount = 20;
+  }
+
+  if (days >= 7) {
+    discount = 50;
+  }
```

Comment:
let's move all these numbers to constants to give them descriptions and make them more understandable